### DISCUSSION TITLE
Is it possible to open multiple instances on Windows?

### DISCUSSION THREAD

**Discussion by Esarak** [jellyfin/jellyfin-desktop#154] [Q&A]
I cannot seem to get jellyfin media player to open multiple simultaneous instances in windows. Is this an architectural limitation or or potentially just a setting that needs to be configured?

**Comment by Esarak** [comment]
It appears that the answer is no, but does anyone know if there is a good reason why?

**Comment by iwalton3** [comment]
That limitation was inherited from Plex Media Player but I have no idea why. I’d have to check but I don’t think JMP listens on any ports so it may be something we can rip out.

**Comment by TheFeelTrain** [comment]
It does open a socket for input according to the log so it could be that. I also find it mildly amusing it means you can't open Plex Media Player and Jellyfin Media Player at the same time. Only one or the other.

**Comment by dcflachs** [comment]
Seeing the change made in v1.7.0 Prerelease 2 I wondered what would happen in if a random number were tacked on the end of the socket name in Paths.cpp. The answer is that it appears to allow multiple instances to run side by side with only minor issues (at least nothing obviously show stopping on the surface). The first notable issue was that only the first instance seemed to have access to stored servers and  credentials, the other instances each needed the info re-added. Second there seems to be a bit of GUI lag on starting videos with multiple instances running, out of curiosity i tried multiple tabs to the same server open in browser (theoretically a similar scenario) with no obvious lag, so not sure where thats from. Finally though i didnt notice anything nor did i dig to deeply, i would wonder what happens with logs and config files with multiple instances. Other than that multiple videos do seem to stream simultaneously without issue.

**Comment by iwalton3** [comment]
I suspect it could be an issue with the local storage only allowing one instance to read it at a time
